common/src/Model/Brush.cpp

```diff
@@ -344,25 +351,25 @@ namespace TrenchBroom {
             }
         }
 
-        BrushFace* Brush::findFace(const Vec3& normal) const {
+        BrushFace* Brush::findFace(const vm::vec3& normal) const {
             for (auto* face : m_faces) {
-                if (face->boundary().normal.equals(normal)) {
+                if (isEqual(face->boundary().normal, normal, vm::constants<FloatType>::almostZero())) {
```

Comment:
The thing is, I want to get rid of the default values for epsilon later on so that you have to specify them at the call site. This is tedious, but it makes you think about which epsilon to use, and why. And then, your isNormalEqual is just just isEqual with a different epsilon (which should be a constant). Or not?